features/view_all_companies.feature

```diff
@@ -17,100 +17,106 @@ Feature: As a visitor,
-      | Company23 | 9074668568     | cmpy13@mail.com        | Stockholm    | Alingsås|
-      | Company24 | 9243957975     | cmpy13@mail.com        | Stockholm    | Alingsås|
-      | Company25 | 9267816362     | cmpy13@mail.com        | Stockholm    | Alingsås|
-      | Company26 | 9360289459     | cmpy13@mail.com        | Stockholm    | Alingsås|
-      | Company27 | 9475077674     | cmpy13@mail.com        | Stockholm    | Alingsås|
+    Given the following simple companies exist:
+      | name      | company_number | email           | region       | kommun  |
+      | Company1  | 0000000001     | cmpy1@mail.com  | Stockholm    | Alingsås|
+      | Company2  | 0000000002     | cmpy2@mail.com  | Västerbotten | Bromölla|
+      | Company3  | 0000000003     | cmpy3@mail.com  | Stockholm    | Alingsås|
+      | Company4  | 0000000004     | cmpy4@mail.com  | Stockholm    | Alingsås|
+      | Company5  | 0000000005     | cmpy5@mail.com  | Stockholm    | Alingsås|
+      | Company6  | 0000000006     | cmpy6@mail.com  | Stockholm    | Alingsås|
+      | Company7  | 0000000007     | cmpy7@mail.com  | Stockholm    | Alingsås|
+      | Company8  | 0000000008     | cmpy8@mail.com  | Stockholm    | Alingsås|
+      | Company9  | 0000000009     | cmpy9@mail.com  | Stockholm    | Alingsås|
+      | Company10 | 0000000010     | cmpy10@mail.com | Stockholm    | Alingsås|
+      | Company11 | 0000000011     | cmpy11@mail.com | Stockholm    | Alingsås|
+      | Company12 | 0000000012     | cmpy12@mail.com | Stockholm    | Alingsås|
+      | Company13 | 0000000013     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company14 | 0000000014     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company15 | 0000000015     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company16 | 0000000016     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company17 | 0000000017     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company18 | 0000000018     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company19 | 0000000019     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company20 | 0000000020     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company21 | 0000000021     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company22 | 0000000022     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company23 | 0000000023     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company24 | 0000000024     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company25 | 0000000025     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company26 | 0000000026     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company27 | 0000000027     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company28 | 0000000028     | cmpy13@mail.com | Stockholm    | Alingsås|
+      | Company29 | 0000000029     | cmpy13@mail.com | Stockholm    | Alingsås|
```

Comment:
Can we rename `Company29`  to `NoBrandingFeePaid` ?  (or something that conveys that meaning)  That would make the purpose and set-up/data of this company very clear.  I had to look through _all_ of the data to confirm that is what this company indeed testing.

Likewise, can `Company28` be renamed to something more meaningful?  I see that there has been a branding fee paid, but I don't know how this quite satisfies the "no members" condition that you're testing in `Scenario: Companies lacking branding payment or members not shown`